liquidation-monitor/src/agent.js

```diff
@@ -0,0 +1,404 @@
+    data.newAccounts = []; // New account from transaction events
+    data.totalNewAccounts = 0;
+
+    // Compound API filter and Comptroller contract
+    const {
+      comptrollerAddress,
+      maxTrackedAccounts,
+      oracleAddress,
+      oneInchAddress,
+      triggerLevels: {
+        maximumHealth,
+        minimumBorrowInETH,
+      },
+    } = config.liquidationMonitor;
+    const comptrollerABI = getAbi(config.liquidationMonitor.comptrollerABI);
+    const oracleABI = getAbi(config.liquidationMonitor.oracleABI);
+    const oneInchABI = getAbi(config.liquidationMonitor.oneInchABI);
+    data.comptrollerAddress = comptrollerAddress;
+    data.comptrollerContract = new ethers.Contract(
+      comptrollerAddress,
+      comptrollerABI,
+      data.provider,
+    );
+    data.oneInchContract = new ethers.Contract(
+      oneInchAddress,
+      oneInchABI,
+      data.provider,
+    );
+    // Unused but available for the future
+    data.oracleContract = new ethers.Contract(
+      oracleAddress,
+      oracleABI,
+      data.provider,
+    );
+    /* eslint-enable no-param-reassign */
+    // #endregion
+
+    // #region Get initial accounts from Compound API
+    // Find total number of results with the first request
+    const initialRequest = buildJsonRequest(maximumHealth, minimumBorrowInETH, 1, 1);
+    const initialResults = await callCompoundAPI(initialRequest);
+    const totalEntries = initialResults.pagination_summary.total_entries;
+
+    // Determine number of pages needed to query. Results vs config limit.
+    const maxEntries = Math.min(maxTrackedAccounts, totalEntries);
+    const maxPages = Math.ceil(maxEntries / 100);
+
+    // Query each page and add accounts. Starting at 1 and including maxPages
+    const foundAccounts = [];
+    // Shorthand Range() function. ( Ex: 5 => [1,2,3,4,5] )
+    const pages = [...Array(maxPages)].map((_, i) => 1 + i);
+    await Promise.all(pages.map(async (page) => {
+      const currentRequest = buildJsonRequest(maximumHealth, minimumBorrowInETH, page, 100);
+      const apiResults = await callCompoundAPI(currentRequest);
+      apiResults.accounts.forEach((currentAccount) => {
+        foundAccounts.push(currentAccount);
+      });
+    }));
+    // #endregion
+
+    // #region Parse Compound data into new objects
+    // Get a full list of token addresses
+    const foundTokens = foundAccounts
+      .map((currentAccount) => currentAccount.tokens.map((token) => token.address)).flat();
+
+    // Use Array to Set to Array to remove duplicates
+    const filteredTokens = Array.from(new Set(foundTokens));
+
+    // Initialize token objects first
+    await Promise.all(filteredTokens.map(async (token) => {
+      await verifyToken(data, token);
+    }));
+
+    // Loop through found accounts
+    foundAccounts.forEach((account) => {
+      // add to tracked accounts
+      /* eslint-disable no-param-reassign */
```

Comment:
It seems like this gets disabled/enabled often, so it should probably be disabled for the entire file or via the `.eslintrc`

Reply:
disabled it in the `.eslintrc` as recommended